api/scpca_portal/templates/readme/additional_terms/research_academic_only.md

```diff
@@ -1,3 +1,35 @@
-### Additional Restrictions
+{% with heading="### Additional Restrictions" text="This dataset is designated as research or academic purposes only." %}
 
-This dataset is designated as research or academic purposes only.
+{% if projects|length > 1 %}
+
+{% with has_additional_restrictions=False %}
+
+{% for project in projects %}
+{% if project.additional_restrictions %}
+{% with has_additional_restrictions=True %}
+{% endwith %}
+{% endif %}
+{% empty %}
+{% endfor %}
+
+{% if has_additional_restrictions %}
+{{ heading }}
+| Project ID | Data Usage Restrictions |
+| :--------- | :---------------------- |
+{% for project in projects %}
+{% if project.additional_restrictions %}
+|[{{ project.scpca_id }}](https://scpca.alexslemonade.org/projects/{{ project.scpca_id }})|{{ project.additional_restrictions }}|
+{% endif %}
+{% endfor %}
+{% endif %}
+
+{% endwith %}
+
+{% elif projects|length == 1 and projects.0.additional_restrictions %}
+
+{{ heading }}
+{{ text }}
+
+{% endif %}
+
+{% endwith %}
```

Comment:
Let's break this up and select the correct template depending on how many projects are passed to the context.


`additional_restrictions.md` / one project
```markdown
{% if project.0.additional_restrictions  === "Research or academic purposes only"%}
### Additional Restrictions
This dataset is designated as research or academic purposes only.
{% endif %}
```
`additional_restrictions_table.md` / multiple projects
```markdown
### Additional Restrictions
| Project ID | Data Usage Restrictions |
| :--------- | :---------------------- |{% for project in projects %}
|[{{ project.scpca_id }}]({{ project.url }})|{{ firstof project.additional_restrictions "No additional restrictions" }}|{% endfor %}
```

Reply:
The readability has been greatly improved, thank you David 👍 !

(See commit at 0dc2d77931389d3f7cb01aed3f561c4ea507abea).